examples/time-series.js

```diff
@@ -115,6 +115,107 @@ try {
 
   console.log('Timeseries info:');
   console.log(tsInfo);
+
+  // Add a new values with LABELS option USING TS.ADD:
+  // https://redis.io/commands/ts.add/
+  const labelTimeSeriesKey1 = "mytimeseries-labels-1";
+  const labelTimeSeriesKey2 = "mytimeseries-labels-2";
+  num = 0; // Reset counter
+  // You can track skipped timestamps due to duplicate policy (OPTIONAL)
+  const skippedTimestamps = [];
+  // Generate a random integer to use as a label value (can demonstrate error, if not a string)
+  const integerLabelValue1 = Math.floor(Math.random() * 1000) + 1;
+  // Sample loop to add 10 values to the timeseries, with labels.
+  console.log("Starting loop 1...");
+  while (num < 10) {
+    await client.ts
+      .add(labelTimeSeriesKey1, currentTimestamp, value + num, {
+        LABELS: {
+          foo: "bar",
```

Comment:
Can we use real world labels rather than `foo`, `bar`, `baz` etc?  These aren't meaningful to pepople.  Perhaps use temperatures as an example and pick random numbers from say -5 to 29 C as the values?

Reply:
Apologies. I overlooked that part in the examples README. Its use is very common on redis.io, so I was in that mindset.

Reply:
Thanks - sadly it is common across Redis and it's something I'm trying to remove over time, but it'll take a while!